python/src/module.py

```diff
@@ -0,0 +1,18 @@
+"""
+File: module.py
+
+Description:
+    Sample module file
+
+Copyright Provizio Ltd., 2022
+Author: Dane Mitrev
+"""
+
+
+class SomeModule(object):
+    def __int__(self) -> None:
+        pass
+
+
+def some_func(some_arg: list) -> None:
```

Comment:
I know it's an example but If you use _list_ and not _List_ from the lib _typing_, you have to use Python 3.9. But only python 3.8 is recommended in the Python Coding Standards for provizio. In that case I am not sure that It would compile correctly.

Reply:
Yes, it's an example but good catch. Although it didn't complain which is strange. I will check.

Reply:
Actually no, even in Python 3.6 arg: list is a correct type hint. 

The difference between py38 and py39 is element types

e.g. you cannot write   some_arg: list[str]  in 3.8 but it's fine in 3.9

So, I will keep it.

Reply:
Ok, thanks for the insight 👍